mysql/tests/common.py

```diff
@@ -11,7 +11,7 @@
 ROOT = os.path.dirname(os.path.dirname(HERE))
 TESTS_HELPER_DIR = os.path.join(ROOT, 'datadog_checks_tests_helper')
 
-MYSQL_VERSION_PARSED = parse_version(os.getenv('MYSQL_VERSION'))
```

Comment:
This lets you run unit tests without setting the version

Reply:
It should always be set by https://github.com/DataDog/integrations-core/blob/27c1070b0c6e2c4211dd22bf4ac0cf3ff566fb83/mysql/tox.ini#L36-L49

Reply:
Not if you run using an IDE. I like to be able to run unit tests with a right click from the IDE